gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -420,5 +420,191 @@ public static ChangeRequestListOption sortOrder(SortingOrder order) {
+   * returned object contains the following read-only fields supplied by the server: id, start time
+   * and status. time, id, and list of name servers. The returned fields can be modified by {@code
+   * ChangeRequestFieldOptions}. Returns null if the zone is not found.
+   *
+   * @return ChangeRequest object representing the new change request or null if zone is not found
+   * @throws DnsException upon failure
+   * @see <a href="https://cloud.google.com/dns/api/v1/changes/create">Cloud DNS Changes: create</a>
+   */
+  ChangeRequest applyChangeRequest(ChangeRequest changeRequest, BigInteger zoneId,
+                                   ChangeRequestOption... options);
+
+  /**
+   * Submits a change requests for applying to the zone identified by name to the service. The
+   * returned object contains the following read-only fields supplied by the server: id, start time
+   * and status. time, id, and list of name servers. The returned fields can be modified by {@code
+   * ChangeRequestFieldOptions}. Returns null if the zone is not found.
+   *
+   * @return ChangeRequest object representing the new change request or null if zone is not found
+   * @throws DnsException upon failure
+   * @see <a href="https://cloud.google.com/dns/api/v1/changes/create">Cloud DNS Changes: create</a>
+   */
+  ChangeRequest applyChangeRequest(ChangeRequest changeRequest, String zoneName,
+                                   ChangeRequestOption... options);
+
+  /**
+   * Retrieves updated information about a change request previously submitted for a zone identified
+   * by ID. Returns null if the zone or request cannot be found.
+   *
+   * <p>The fields to be returned using {@code ChangeRequestFieldOptions}.
+   *
+   * @throws DnsException upon failure
+   * @see <a href="https://cloud.google.com/dns/api/v1/changes/get">Cloud DNS Chages: get</a>
+   */
+  ChangeRequest getChangeRequest(ChangeRequest changeRequest, BigInteger zoneId,
+                                 ChangeRequestOption... options);
+
+  /**
+   * Retrieves updated information about a change request previously submitted for a zone identified
+   * by name. Returns null if the zone or request cannot be found.
+   *
+   * <p>The fields to be returned using {@code ChangeRequestFieldOptions}.
+   *
+   * @throws DnsException upon failure
+   * @see <a href="https://cloud.google.com/dns/api/v1/changes/get">Cloud DNS Chages: get</a>
+   */
+  ChangeRequest getChangeRequest(ChangeRequest changeRequest, String zoneName,
+                                 ChangeRequestOption... options);
+
+  /**
+   * Lists the change requests for the zone identified by ID that were submitted to the service.
+   *
+   * <p>The sorting key for changes, fields to be returned, page and page tokens can be specified
+   * using {@code ChangeRequestListOptions}. Note that the only sorting key currently supported is
+   * the timestamp of submitting the change request to the service.
```

Comment:
Just to be sure I understand. [Here](https://cloud.google.com/dns/api/v1/changes/list#sortBy) I read that:

> Sorting criterion. The only supported value is change sequence. 

Also, how can the sorting key/criterion be specified? I can not find an option in `ChangeRequestListOption`